src/Nethermind/Nethermind.Evm/Tracing/ParityStyle/ParityTraceTypes.cs

```diff
@@ -2,17 +2,23 @@
 // SPDX-License-Identifier: LGPL-3.0-only
 
 using System;
+using System.ComponentModel;
 
-namespace Nethermind.Evm.Tracing.ParityStyle
+namespace Nethermind.Evm.Tracing.ParityStyle;
+
+[Flags]
+public enum ParityTraceTypes
 {
-    [Flags]
-    public enum ParityTraceTypes
-    {
-        None = 0,
-        VmTrace = 1,
-        StateDiff = 2,
-        Trace = 4,
-        Rewards = 8,
-        All = 15,
-    }
+    [Description("None.")]
+    None = 0,
+    [Description("Traces EVM data.")]
+    VmTrace = 1,
+    [Description("Traces the state and storage.")]
+    StateDiff = 2,
+    [Description("Traces actions and block receipts.")]
+    Trace = 4,
+    [Description("Traces block rewards.")]
+    Rewards = 8,
```

Comment:
@LukaszRozmej Could you please review these descriptions?

Reply:
from: https://openethereum.github.io/JSONRPC-trace-module

```suggestion
    [Description("None.")]
    None = 0,
    [Description("Virtual Machine execution trace. Provides a full trace of the VM’s state throughout the execution of transactions at each op-code, including for any subcalls.")]
    VmTrace = 1,
    [Description("State difference. Provides information detailing all altered portions of the Ethereum state made due to the execution of transactions.")]
    StateDiff = 2,
    [Description("Transaction trace including subcalls.")]
    Trace = 4,
    [Description("Includes block rewards in trace when tracing full blocks.")]
    Rewards = 8,
```